packages/e2e/README.md

```diff
@@ -106,22 +113,27 @@ dependencies {
 To e2e test on a local Android device (emulated or physical):
 
 ```
-./gradlew connectedAndroidTest -Ptarget=`pwd`/../test_driver/<package_name>_e2e.dart
+./gradlew app:connectedAndroidTest -Ptarget=`pwd`/../test_driver/<package_name>_e2e.dart
```

Comment:
The suffix should be `_e2e.dart` or `_e2e_test.dart` as above?

Reply:
The suffix here should be _e2e.dart. The _e2e_test.dart suffix is reserved for the driver runner (which I hope will be unnecessary in the future).